hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2053,6 +2053,14 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.scm.ratis.minimum.timeout</name>
```

Comment:
We need to remove this from ozone-default.xml and SCMConfigKeys

And we should change below code where we are using config from SCMConfigKeys to SCMHAConfiguration. And also change it to 30000ms


```
    final long requestTimeout = scm.getConfiguration()
        .getTimeDuration(ScmConfigKeys.OZONE_SCM_RATIS_REQUEST_TIMEOUT_KEY,
            ScmConfigKeys.OZONE_SCM_RATIS_REQUEST_TIMEOUT_DEFAULT,
            TimeUnit.MILLISECONDS);
```

Should be changed as 
```
    final long requestTimeout =
        scm.getConfiguration().getObject(SCMHAConfiguration.class)
            .getRatisRequestTimeout();
```

cc @bshashikant to check if it okay to use this here also.

Reply:
its okay. let's do this change. Also, let's change the default to 10s

Reply:
Previously we used to use 30s, now reduce that to 10s?

```
  public static final String OZONE_SCM_RATIS_REQUEST_TIMEOUT_DEFAULT
      = "30s";
```

Reply:
30s is fine

Reply:
I have modified the code, now it will use the config from SCMHAConfiguration and also have set the default value to 30 seconds.